test/functionalTest/cases/0989_GET_v2_entity_attribute/GET_v2_entity_attribute_too_many.test

```diff
@@ -0,0 +1,142 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Entity attribute -- Too many entities
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v1/updateContext, creating 5 entities
+# 02. GET /v2/entities/E1/attrs/A0
+#
+
+echo "01. POST /v1/updateContext"
+echo "=========================="
+payload='{
+  "contextElements": [
+    {
+      "id":   "E1",
+      "type": "T1",
+      "attributes": [
+          {
+              "name": "A0",
+              "type": "number",
+              "value": "1"
+          }
+      ]
+    },
+    {
+      "id":   "E1",
+      "type": "T2",
+      "attributes": [
+          {
+              "name": "A0",
+              "type": "number",
+              "value": "2"
+          }
+      ]
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+echo "02. GET /v2/entities/E1/attrs/A0"
+echo "=================================="
```

Comment:
Line with '=' should be adjusted to the length of the step description line (i.e. delete 2 final '=').